src/components/trie/utils.ts

```diff
@@ -52,7 +52,7 @@ export function trieToTreeUI(
   return {
     name: hash.toString(),
     attributes: {
-      key: leaf.getKey().toString(),
+      keyHash: leaf.getKey().toString(),
```

Comment:
```suggestion
      key: leaf.getKey().toString(),
```

Key is fine. I know it "looks" like a hash, but calling it "key hash" implies that the "key" data is hashed again, and that's not correct.